src/jit/morph.cpp

```diff
@@ -2865,19 +2865,7 @@ void Compiler::fgInitArgInfo(GenTreeCall* call)
     // We are allowed to have a Fixed Return Buffer argument combined
     // with any of the remaining non-standard arguments
     //
-    if (call->IsUnmanaged() && !opts.ShouldUsePInvokeHelpers())
```

Comment:
@dotnet/jit-contrib Do you still maintain the .NET Framework of the JIT for testing purposes? This change would break it.

Reply:
We do, informally. I think at least @briansull tries to keep it working in our internal mirror.

Reply:
I spoke to @briansull about this and he is aware. He didn't seem too bothered by it, but it was merely a conversation about my plans rather than a code example. Please let me know if I should bring this back.

Reply:
Probably best for now to place this code in an ifdef:

```
#if !defined(FEATURE_CORECLR)
    if (call->IsUnmanaged() && !opts.ShouldUsePInvokeHelpers())
```